internal/policies/mount/helper_test.go

```diff
@@ -0,0 +1,37 @@
+package mount
+
+import (
+	"fmt"
+
+	"github.com/ubuntu/adsys/internal/policies/entry"
+)
+
+// GetEntries generates entries to be used in tests.
+func GetEntries(nEntries, nErrored, nValues, nDuplicated int, sep []string) []entry.Entry {
+	entries := []entry.Entry{}
+	c := 0
+	for i := 0; i < nEntries; i++ {
+		if i < nErrored {
+			e := entry.Entry{Err: fmt.Errorf("this entry has an error")}
+			entries = append(entries, e)
+			continue
+		}
+
+		e := entry.Entry{Key: fmt.Sprintf("mount/%d", i)}
+		dup := 0
+		value := "protocol://domain.com/mount-%d-path"
```

Comment:
I would put a mix of protocols and fictive subpath to illustrate a real entry type. That way, by looking at the golden files, we know that we support multiple paths hierachy. (Not all needs to be like that, but some).